system-variables.md

```diff
@@ -2127,6 +2135,30 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - Range: `[0, 9223372036854775807]`
 - This variable is used to limit the maximum number of requests TiDB can send to TiKV at the same time. 0 means no limit.
 
+### tidb_super_read_only <span class="version-mark">New in v5.3.1</span>
+
+> **Warning:**
+>
+> Prior to v5.3.1/v5.4.1, there was a serious bug with this variable. Please use this variable in and after v5.3.1/v5.4.1.
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Type: Boolean
+- Default value: `OFF`
+- This variable controls the read-only status of the entire cluster. When the variable is `ON`, all TiDB servers in the entire cluster are in the read-only mode. In this case, TiDB only executes the statements that do not modify data, such as `SELECT`, `USE`, and `SHOW`. For other statements such as `INSERT` and `UPDATE`, TiDB rejects executing those statements in the read-only mode.
+- Enabling the read-only mode using this variable only ensures that the entire cluster finally enters the read-only status. If you have changed the value of this variable in a TiDB cluster but the change has not yet propagated to other TiDB servers, the un-updated TiDB servers are still **not** in the read-only mode.
+- When this variable is enabled, the SQL statements being executed are not affected. TiDB only performs the read-only check for the SQL statements **to be** executed.
```

Comment:
This part is no longer true. See https://github.com/pingcap/tidb/pull/35487

```suggestion
- TiDB checks the read-only flag before SQL statements are executed. Since v6.2.0, the flag is again checked before SQL statements are committed. This helps prevent a case where long running auto_commit statements could modify data after the server has been placed in read-only mode.
```